skbio/io/format/tests/test_clustal.py

```diff
@@ -25,19 +25,19 @@
 from skbio.io import ClustalFormatError
 
 
-class CustomSequence(IUPACSequence):
+class CustomSequence(GrammaredSequence):
     @classproperty
-    @overrides(IUPACSequence)
+    @overrides(GrammaredSequence)
```

Comment:
You added `default_gap_char` to previous `CustomSequence` classes in the unit tests. Is `default_gap_char` not abstract?